Client/ContentBlocker/TabContentBlocker.swift

```diff
@@ -56,8 +56,8 @@ class TabContentBlocker {
         NotificationCenter.default.addObserver(self, selector: #selector(notifiedTabSetupRequired), name: .contentBlockerTabSetupRequired, object: nil)
     }
 
-    func scriptMessageHandlerName() -> String? {
-        return "trackingProtectionStats"
+    func scriptMessageHandlerNames() -> [String]? {
+        return ["trackingProtectionStats"]
```

Comment:
Is this specific to Autofill that we need to make the change everywhere?

Reply:
Hello! Curious about this change as well, why do we need to add multiple handlers for a single script?

Reply:
Yes for now this is specific to Autofill. Initially I thought it would be a good idea to have it as an [extension](https://github.com/mozilla-mobile/firefox-ios/pull/17601/files#diff-c388a2f2d3107fbda12b4ed02b1b0af39db749b91e1bb8f54a4119014fa893d6R35-R45), but changing the definition to return an array seems more consistent.

Reply:
@lmarceau for form autofill we have two cases address and credit card. We want to reuse the same script for both. If we don't know before hand which type of message it is we will have to parse JSON from js check the type and then parse it according to some predefined swift type. This simplifies things a bit as we know before hand which type to use for parsing.